releng/release-notes-next/expanduser-with-copyout.bugfix

```diff
@@ -0,0 +1,4 @@
+Previous versions of Mock mistakenly expanded every `~` occurrence
+(tilde character) in the specified source path with `--copyout`.  So
+files `~/foo~bar.txt` were searched on path `/builddir/foo/builddirbar.txt`
+instead of just `/biulddir/foo~bar.txt`.  Fixes [rhbz#2239035][].
```

Comment:
There is a minor typo in the path in this line:

`/biulddir/foo~bar.txt` instead of `/builddir/foo~bar.txt`

Reply:
Thanks for working on this!

Reply:
Heh, do you mean the additional `just` word?  Sorry my eyes are tired today :D

Reply:
@praiskup No. `builddir` is spelled wrong with `i` coming before `u`: `biuld`